src/teaching-element/Assessment/DragDrop.vue

```diff
@@ -40,7 +40,9 @@
                 v-if="!disabled"
                 @click="removeFromBox(id, response)"
                 class="btn btn-close">
-                <span :class="config.removeClass"></span>
+                <slot name="dragDropRemoveIcon">
+                  <span :class="config.removeClass"></span>
```

Comment:
This component used a prop to set a custom icon through classes. A slot allows the setting of an icon by sending a tag that can have the classes previously set through props. So I do not see a reason for allowing both. We are already going to have breaking changes so this can be an additional one.

Reply:
Not sure what you are suggesting here?

While we are here I think the `removeClass` computed method can be removed

Reply:
That the config option for setting class be removed because the slot can be used instead. And yes, the computed should be removed.

Reply:
Done!